src/ssh/azext_ssh/_params.py

```diff
@@ -19,6 +19,9 @@ def load_arguments(self, _):
         c.argument('cert_file', options_list=['--certificate-file', '-c'],
                    help='Path to a certificate file used for authentication when using local user credentials.')
         c.argument('port', options_list=['--port'], help='SSH port')
+        c.argument('ssh_client_folder', options_list=['--ssh-client-folder'],
+                   help='Path to folder that contains ssh executables (ssh.exe, ssh-keygen.exe, etc). '
+                   'Default to ssh pre-installed if not provided.')
         c.positional('ssh_args', nargs='*', help='Additional arguments passed to OpenSSH')
 
     with self.argument_context('ssh config') as c:
```

Comment:
Mismatch.
Line 28, config_path is defined as file path but ssh_config() treats it as a folder.

Reply:
Config path is not treated as a folder. There is some logic in ssh_config() that checks if the folder exists, but that takes the dirname of the config_path. Is that what you are referring to?